src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -104,6 +112,41 @@ public async Task BackgroundUpdateAdvertisingManifestsWhenRequiredAsync()
             }
         }
 
+        private void WriteUpdatableWorkloadsFile()
```

Comment:
Once you get an update advertised, and then you run `dotnet workload update`, is there anything that turns off the update advertising?  I'm not seeing where this file is deleted or updated once you update workloads.